python-threatexchange/threatexchange/exchanges/impl/ncmec_api.py

```diff
@@ -39,14 +39,16 @@ class NCMECCheckpoint(
 
     # The biggest value of "to", and the next "from"
     get_entries_max_ts: int
+    next_fetch: t.Optional[str] = ""
```

Comment:
blocking: Hmm, not sold on this name. NCMEC calls these "[paging urls](https://report.cybertip.org/hashsharing/v2/documentation/?fbclid=IwZXh0bgNhZW0CMTEAAR3z8y_XUaLoQqTw-SlwsYrwMmMpz1xzXoOKgG4vy30xLEIDJxI22qJaiuM_aem_rN1itRmqgz3skMyJ-2q11Q#pagination)" so maybe paging_url?
blocking; Add a comment explaining what this variable represents.

nit: Since the default value is empty string, suggest making this non-`Optional` and just using that.